hypervisor/src/kvm/mod.rs

```diff
@@ -2620,12 +2650,63 @@ impl cpu::Vcpu for KvmVcpu {
             .map_err(|e| cpu::HypervisorCpuError::SetDebugRegs(e.into()))
     }
 
+    #[cfg(target_arch = "aarch64")]
+    fn vcpu_get_finalized_features(&self) -> i32 {
+        kvm_bindings::KVM_ARM_VCPU_SVE as i32
+    }
+
+    #[cfg(target_arch = "aarch64")]
+    fn vcpu_set_processor_features(
+        &self,
+        vm: &Arc<dyn crate::Vm>,
+        kvi: &mut VcpuInit,
+        id: u8,
+    ) -> cpu::Result<()> {
```

Comment:
It appears to be a refactoring, while it actually introduced a functional change. Please double check if it is properly used as you intended to.

Reply:
I have verified that the changes results in the same old state as the previously written code and all the ARM related unit tests and integration tests are passing.